src/frontend/src/flows/dappsExplorer/index.ts

```diff
@@ -58,7 +61,11 @@ const dappTemplate = ({
       rel="noopener noreferrer"
     >
       <div class="c-action-list__icon" aria-hidden="true">
-        <img src=${logo} alt=${name} loading="lazy" />
+        <img
+          src=${logo.replace("/img/showcase/", "/icons/")}
```

Comment:
second time I notice `logo.replace("/img/showcase/", "/icons/"`, probably worth mapping the icons or a function

Reply:
One of these needs to go, that's my bad. Will remove after this PR is merged (got another one ready)